rq/logutils.py

```diff
@@ -5,14 +5,16 @@
 import logging
 
 from rq.utils import ColorizingStreamHandler
+from rq.defaults import (DEFAULT_LOGGING_FORMAT,
+                         DEFAULT_LOGGING_DATE_FORMAT)
 
 
-def setup_loghandlers(level):
+def setup_loghandlers(level, datefmt=DEFAULT_LOGGING_DATE_FORMAT,
```

Comment:
One last minor detail, could you please change the `datefmt` to `date_format` and `logfmt` to `log_format`? This should keep the variable naming consistent across RQ's code base.

Reply:
Sorry my bad. Fixed now